extensions/amp-story/0.1/system-layer.js

```diff
@@ -148,17 +163,19 @@ export class SystemLayer {
    */
   addEventHandlers_() {
     // TODO(alanorozco): Listen to tap event properly (i.e. fastclick)
-    this.exitFullScreenBtn_.addEventListener(
-        'click', e => this.onExitFullScreenClick_(e));
-
-    this.closeBookendBtn_.addEventListener(
-        'click', e => this.onCloseBookendClick_(e));
-
-    this.muteAudioBtn_.addEventListener(
-        'click', e => this.onMuteAudioClick_(e));
-
-    this.unmuteAudioBtn_.addEventListener(
-        'click', e => this.onUnmuteAudioClick_(e));
+    this.root_.addEventListener('click', e => {
```

Comment:
I think we can simplify this a bit going forward, perhaps by using CSS grid for the layout so we can move elements around without having to change their ancestors.  That will allow us to have one button for each purpose across our different UIs (i.e. we won't need to duplicate the audio controls).

Filed #11640 to track and we can refactor later.

Reply:
Makes sense. I was optimizing for simplicity to have it implemented quickly. I think using the css grid would work well